Move Zeroes/main.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DefaultNamespace
+{
+    public class main
+    {
+        public void MoveZeroes(int[] nums)
+        {
+            int tem = 0;
+            for (int i = 0; i < nums.Length; i++)
```

Comment:
Let's try to have a pointer for zero value index and optimize the solution to O(n)